assets/src/edit-story/app/story/actions/useSaveStory.js

```diff
@@ -73,6 +74,10 @@ function useSaveStory({ storyId, pages, story, updateStory }) {
           updateStory({ properties });
 
           refreshPostEditURL();
+
+          setIsFreshlyPublished(
+            story.status !== 'publish' && post.status === 'publish'
```

Comment:
Ah, forgot about scheduled content.

I feel like I need to pass the post status to `setIsFreshlyPublished` so the dialog can have the appropriate message (`story successfully scheduled!`)

> Wondering if this should be asked just once for the post? Or if the user already has chosen to add it to a post at some point previously, maybe it shouldn't ask again?

Hmm interesting idea, will need UX consideration. And a way to store that information if necessary.

Reply:
The name of this function doesn't seem correct. I made a story publish, then draft and the published it again. I still saw this message. What should happen in this use case? Should have a little bit of post meta, to say it also has a post linked to it?

Reply:
So... essentially the same question as Miina's.

It's a UX question we'll need to discuss on the issue.

Reply:
As revealed at https://github.com/google/web-stories-wp/issues/1118#issuecomment-642474545, this will look much different for stable.

For stable, flows like future->publish or publish->draft->publish will be mostly irrelevant, as the new flow will be way less intrusive.

For now, what I will change is that future->publish or publish->future won't trigger the message.

Everything beyond that won't be necessary since, again, this will be completely revamped for Stable.